descriptor.md

```diff
@@ -161,6 +163,16 @@ Implementations MAY implement SHA-512 digest verification for use in descriptors
 When the _algorithm identifier_ is `sha512`, the _encoded_ portion MUST match `/[a-f0-9]{128}/`.
 Note that `[A-F]` MUST NOT be used here.
 
+#### BLAKE3
+
+[BLAKE3][blake3] is a high performance, highly parallelizable, collision-resistant hash function which [is more performant][blake3-vs-sha2] than
+[SHA-256][rfc4634-s4.1].
+The hash output length MUST be 256 bits.
```

Comment:
I'm not familiar with BLAKE3, but going through [the spec][1], it seems like we might also need to specify [the `hash` hashing mode][2]?  Or is it sufficiently obvious from the context that there's no provision for supplying a key or other input, making the `keyed_hash` and `derive_key` modes unfeasible?

[1]: https://github.com/C2SP/C2SP/blob/BLAKE3/v1.0.0/BLAKE3.md
[2]: https://github.com/C2SP/C2SP/blob/BLAKE3/v1.0.0/BLAKE3.md#11--hashing-modes

Reply:
IMO it's pretty obvious that this is and has to be unkeyed (as you said, there's no way to specify a key).

Reply:
Similar to sha1 or sha2  right?

The purpose of the hash here is content-addressability (with reason non-collision guarantees)

You can still build a keyed hash out of these - https://datatracker.ietf.org/doc/html/rfc2104